shopfloor_reception_mobile/static/src/scenario/reception.js

```diff
@@ -153,6 +153,7 @@ const Reception = {
                         <v-row align="center">
                             <v-col class="text-center" cols="12">
                                 <btn-back/>
+                                <cancel-button/>
```

Comment:
```suggestion
                        </v-col>
                        <v-col class="text-center" cols="12">
                                <cancel-button/>
```

Reply:
A own v-col would be great i think. 
![image](https://github.com/OCA/wms/assets/78783475/e628ec3f-c05a-4fc1-87ee-fb4ef2ed8722)

With suggestion, it would look like this. 
![image](https://github.com/OCA/wms/assets/78783475/3dd8ab47-29fa-4a05-9a25-f23a051ace95)

Reply:
Do we need both back & cancel ?

Reply:
I have done the change to have the button spaced out.